common/print.c

```diff
@@ -84,26 +85,109 @@ p11_print_word (FILE *fp,
 	fprintf (fp, "m%s\033[0m", string);
 }
 
+/* List of colors used for nested section headers */
+static p11_color header_colors[] = {
+	P11_COLOR_BLUE,
+	P11_COLOR_GREEN
+};
+
+/* Default indentation */
+#define INDENT "    "
```

Comment:
nit: I kind of feel like this is unnecessary. Its a file local constant which is only ever used inside `print_indent` function ones and never anywhere else. In this case I would just have it inside `print_indent`